### PR TITLE
Added prep work for Wildfly tests

### LINKED ISSUE #2449: Tweetest Prep work for wildfly tests
**👾 Description of the issue**

We run external third-party application tests at this project.  They are contained within the external directory.  We would like to improve how we run these tests.  The first step to improving how we run them is to add some documentation and to pull out the individual test properties that we use so that they are stored in the same directory as the test.

This issue is to add a README.md and a test.properties file to the /external/wildfly test directory.  As part of this prep work, since we already generate the Dockerfile, please also remove the static https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/external/wildfly/dockerfile/Dockerfile.  

The contents of the README file can be simply some instructions for how to run the test locally.  Use with the instructions from https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/external/README.md#running-external-tests-locally and modify Steps 7 and 9 to reference wildfly and wildfly_test specifically.

The contents of the test.properties file will be a key=value format, copying the information from https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/external/common_functions.sh#L453-L465 to populate it.

Adding these files will not change the current way we run these tests.  A separate change will be made to reference the test.properties file, at which point the contents must match those in the common_functions.sh script.  

Please take the time to try and run the test locally (using the instructions you are adding in the README file).  Some of the external tests are failing.  When you run it locally, you may find that to be the case.  The reviewer can help to advise you if there are any further steps you can take to try and debug the pre-existing failure, if it exists.

**📋 Step by Step**

To solve this issue and contribute a fix you should check the following step-by-step list. A more detailed documentation of the workflow can be found [here](https://github.com/firstcontributions/first-contributions/blob/master/README.md).  Note: that you do not need to add yourself to a Contributors.md file as outlined in the detailed instructions.

- [ ] Claim this issue: Comment below.
- [ ]  Fork the repository in github by simply clicking the 'fork' button.
- [ ]  Check out the forked repository
- [ ]  Create a feature branch for the issue. We do not have any naming definition for branches.
- [ ]  Commit your changes.
- [ ]  Start a Pull Request.
- [ ]  Done 👍 Ask in comments for a review :)
- [ ]  If the reviewer find some missing pieces or a problem they will start a discussion with you and describe the next steps how the problem can be solved.
- [ ]  You did it 🎉 We will merge the fix in the master branch.
- [ ]  Thanks, thanks, thanks for being part of this project as an open source contributor ❤️

**🤔❓ Questions**

If you have any questions just ask us directly in this issue by adding a comment. You can join our community chat at [Slack](https://adoptopenjdk.net/slack.html). Next to this you can find a general manual about open source contributions [here](https://github.com/firstcontributions/first-contributions/blob/master/README.md).

### PR DESCRIPTION
**Pull request by SAY-droid427**
Fixes #2449
Completed the prep- work for wildfly tests
- Removed the static dockerfile
- Added the README.md
- Added the tests.properties
- 
Signed-off-by: SAY-droid427 <sayanimallick0218@gmail.com>

### CODE REVIEW

**Comment by SAY-droid427** [comment]
@smlambert @sophia-guo Please review this PR.

**Review by sophia-guo — APPROVED** [review]
LGTM

**Comment by sophia-guo** [comment]
Grinder: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/14/

File: external/wildfly/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# External Wildfly Tests
+
+Wildfly tests are part of the external third-party application tests that help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue [#172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests. Wildfly tests are pulled from the [wildfly-test-suite](https://github.com/wildfly/wildfly.git) repository.
```
Comment (smlambert) [inline external/wildfly/README.md:3]: In last sentence, change wildfly-test-suite to wildfly (since we are using the wildfly repo)

**Review by smlambert — COMMENTED** [review]
A minor change requested, but otherwise LGTM @SAY-droid427

**Comment by smlambert** [comment]
Not related to this PR, but noticing other .tap files in 2 Grinder links today:
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/14/tapResults/

File: external/wildfly/README.md (hunk 1 above)
Comment (SAY-droid427) [reply external/wildfly/README.md]: @smlambert I have changed the last line to wildfly in place of wildfly-test-suite.

**Review by smlambert — APPROVED** [review]

**Pull request merged**